charts/cryostat/auth_proxy_htpasswd

```diff
@@ -0,0 +1 @@
+user:$2y$05$.p1/68oBWKX1FhAtZOArYOoNoMqsB4yuUNXGOeSATHPq3geKqEaZS
```

Comment:
It doesn't seem like the authproxy is actually using this `htpasswd` file. When I `helm install` now I'm still able to just open the `route` and go straight into the Cryostat Web UI without having to authenticate to the proxy.

I suspect the problem is that the `service` and/or `route` are still being defined so that traffic is flowing directly into the Cryostat container, whereas it should now be flowing through the authproxy conntainer (which will then proxy to Cryostat and the other containers).

Reply:
Yep - I think `service.yaml` at least needs adjusting so that it points to `8080` (authproxy) instead of `8181` (cryostat3).

Reply:
Sorry - that should be `4180` now with the port binding change, not `8080`.